### ISSUE TITLE
Brew: calling bottle unneeded is deprecated!

### ISSUE THREAD

**Issue by wmorgue** [benbjohnson/litestream#244]
Hi 👋🏻
Run brew upgrade today on macOS and see the following warning:

```bash
$ brew upgrade
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the benbjohnson/litestream tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/benbjohnson/homebrew-litestream/litestream.rb:6
``` 

macOS 11.6
litestream version v0.3.6
Homebrew 3.2.17-42-gb77b08c
Homebrew/homebrew-core (git revision feaadeedef7; last commit 2021-10-20)
Homebrew/homebrew-cask (git revision e024e53626; last commit 2021-10-20)

![Screen Shot 2021-10-20 at 11 19 48](https://user-images.githubusercontent.com/24982084/138056675-fa376de4-cb58-4191-b4ca-1eac46266791.png)

**Comment by benbjohnson** [comment]
@wmorgue  Sorry for the late reply but thank you for reporting it. I'll get this fixed up for the `v0.4.0` release.

**Comment by benbjohnson** [comment]
Looks like it doesn't need to wait for a new version release. I updated the formula and it seems to work without warning now. https://github.com/benbjohnson/homebrew-litestream/commit/0c17ae662bb7e6f363ef1e6df2981393b6e0c094
